src/tests/mod.rs

```diff
@@ -167,13 +179,38 @@ impl TestToken<'_> {
         }
     }
 
+    #[allow(unreachable_code)]
+    fn get_default_db(name: &str) -> (&'static str, String) {
+        /* in order of preference, the first wins */
+        #[cfg(feature = "nssdb")]
+        {
+            let ret = format!("configDir={}/{}/", TESTDIR, name);
+            return (storage::nssdb::DBINFO.dbtype(), ret);
```

Comment:
Do we actually need to get this string by making modules public ?
At the end the dbtype() is nothing else than the db name which is what we are using to select which storage option we are going to fetch this same string from...
I'd like to keep the modules private to the storage tree, and accessory function would work as well if needed()

Reply:
Yeah by accessor I meant a function exposed in mod.rs ... but no biggie, it is ok as is for now